usage_policy.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: usagepolicy
+title: Telemetry Data Collection Notice
+subtitle: 
+permalink: /usage/policy
```

Comment:
The image above is outdated I think as we made it point to `usage` (I created a PR and you approved it)?

Reply:
yes the image was made before the PR was merged.

/usage is for the general landing page, o that is a link to /usage/policy that has the legal verbage.